### DISCUSSION TITLE
New option for SlashCommands builder. .addAttatchmentOption()

### DISCUSSION THREAD

**Discussion by Danii47** [discordjs/discord.js#7585] [Q&A] [answered]
In such a way that there is the addStringOption, addUserOption etc., there could be an option like addAttatchmentOption to be able to force the user to attach a file. It would be very useful when creating slash commands as there is no way to get the attached files when using a slash command.

**Comment by almeidx** [comment, marked as answer]
Already available on the `dev` release https://github.com/discordjs/discord.js/blob/a8321d8026df2e6a09d867939986bf77f894f3a8/packages/builders/src/interactions/slashCommands/mixins/SharedSlashCommandOptions.ts#L57-L66

**Reply by Danii47** [reply]
Oh, I didn't know that option existed, in fact I was looking for it, thank you very much :)
